client/lib/balena.ts

```diff
@@ -1,4 +1,4 @@
-import { BalenaSDK, DeviceTag } from 'balena-sdk';
+const { BalenaSDK, DeviceTag } = require('balena-sdk')
```

Comment:
Why make these changes in this PR? They seem unrelated to the config package, at least make it a separate commit.

Reply:
ready to review